src/Operation/DatabaseCommand.php

```diff
@@ -92,7 +92,6 @@ public function __construct($databaseName, $command, array $options = [])
      * Execute the operation.
      *
      * @see Executable::execute()
-     * @param Server $server
      * @return Cursor
      */
     public function execute(Server $server)
```

Comment:
I know `Executable::execute()` doesn't define a return type hint (BC yada yada), but I imagine we can still use return type hints in the implementations per covariance if you like.

Reply:
Similar to the situation with `Explainable`, adding a return type here would constitute a BC break.